plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -572,14 +603,15 @@ fu_usi_dock_mcu_device_write_firmware_with_idx(FuUsiDockMcuDevice *self,
 					 sizeof(val),
 					 error))
 		return FALSE;
```

Comment:
I think you can get rid of this initial one as well, no?

Reply:
@victor-cheng I'm prepping the new fwupd release for tomorrow, so if you want this in the tarball then can you fix this up today please. If timing is not important then no panic.